installer/scripts/omsadmin.sh

```diff
@@ -325,6 +326,35 @@ set_proxy_setting()
     fi
 }
 
+is_scom_port_open()
+{
+    OIFS=$IFS
+# Ignore formatting. It was required for IFS to take new line on both bash and sh
+IFS='
+'
+    for line in `cat $OMI_CONF_FILE | grep httpsport`
+    do
+        echo $line | grep -E '^#.*$' > /dev/null 2> /dev/null
+        is_comment=$?
+        if [ $is_comment -eq 0 ]; then
+            continue
+        fi
+        # 1st uncommented line is checked for port
+        # Replace = and , with space.
+        # e.g httpsport=0,1270 should become httpsport 0 1270
+        m_line=`echo $line | sed "s/[=,]/ /g"`
```

Comment:
Well, this is one way to do it. It might have been just as easy to extend `omiconfigeditor`, but that's neither here nor there. But this would work. I was afraid to use `egrep` for this (but did think about that). But then I forgot that `grep -E` does the same thing.

This code is definitely flawed. You'll match on 1270 on a line, even if it's on some other tag, and that's a problem. I would probably change the line at line 339 to continue if we don't begin with `<optional-spaces>httpsport<optional-spaces>=`. If we don't begin with that, then we definitely don't care about this line. Or maybe just see if we begin with `httpsport=`; it would be best to look at [OMI](https://github.com/Microsoft/omi/blob/master/Unix/server/server.c#L774) and see how picky it is about parsing. I'd need to dig deeper into `Conf_Read()` to fully understand that.

Also, did we have a prior dependence on the `sed` program? OMS is super-picky about not picking up dependencies on programs, as some minimal installations of Linux may not include tools that I thought were everywhere (tar, etc). I wouldn't want to inadvertently pick up a dependency on `sed` if we didn't already have it.

That was part of my reason for suggesting changes to `omiconfigeditor`; that would allow us to avoid new dependencies. But I just want something that works and satisfies OMS's needs.